Projects/hGraphQL/GraphQL/MemberDetails.graphql

```diff
@@ -0,0 +1,9 @@
+query MemberDetails {
+  member {
+    id
+    firstName
+    lastName
+    email
+    phoneNumber
+  }
```

Comment:
These are the details that will be posted in Slack. No previous query was fetching all this in one go. So had to add a new one